mgmtsystem/ec2.py

```diff
@@ -16,6 +21,12 @@
 )
 
 
+def regions(regionlist, regionname):
```

Comment:
nice first step, to get where i asked please replace the `regionlist` with the `module` and use `module.regions()` in there

also naming it `_region_by_name` should help to avoid public usage